### ISSUE TITLE
Ionic Package for iOS not working

### ISSUE THREAD

**Issue by dmastag** [driftyco/ionic-cloud#55]
Since this weekend, everytime I build using Ionic Package I am getting this error in the application loader.
Tried this out with 2 of my previous working apps and with a brand new ionic 2 app `ionic start pushtest --v2`

![image](https://cloud.githubusercontent.com/assets/5745805/18856813/7a060440-8488-11e6-81c0-3c6fa319937d.png)

**Comment by ericb** [comment]
Are you able to submit something built locally? This appears to be caused by certain builds of (mac sierra)[http://stackoverflow.com/questions/37838487/error-itms-90167-no-app-bundles-found-in-the-package], but I'm not sure if this is caused by the version submitting the .ipa or the version building it. We are not yet running sierra on our package servers.

**Comment by dmastag** [comment]
Yes, I am able to submit locally build ones. Had to use xcode and it was working.
Some other guys on the slack channel reported the same issues and other reports also.

Are there some changes on the package servers?
